### PR TITLE
Skip calibration RBs

### LINKED ISSUE #451: Ensure that we are skipping RB 0
Issue raised by: [developer/user]

### What?
It would appear that RB 0 is no longer being skipped. I suggest this is because it existed in the EndOfRunMonitor code which was dropped in favour of the new ingestion solution. 
As such we should look to add this functionality back in 

### Where?
Queue Processor? 

### How?
Noticed with pre cycle runs 

### Reproducible?
Yes

### How to test the issue is resolved
* Manually submit runs that are of RB 0 or below and ensure they turn up in the database as skipped. 
* The database and the web app should report these as skipped and should display a sensible warning message

### PR DESCRIPTION
**Pull request by ElliotAOram**
### Summary of work
Ensure that RB0 or less AND String RBs are skipped and added to the skipped queue (not just never run)

### How to test your work
Manually submit an RB 0, ensure it shows up in the webapp as skipped and has an explanation 

Fixes #451

**Before merging ensure the release notes have been updated**

### CODE REVIEW

**Comment by ElliotAOram** [comment]
Need to do my own manual testing + release notes before merge

File: queue_processors/queue_processor/queue_processor.py [hunk 1]
```diff
@@ -216,6 +205,17 @@ def data_ready(self):
         self._data_dict['reduction_script'] = reduction_script
         self._data_dict['reduction_arguments'] = arguments
 
+        # Make sure the RB number is valid
```
Comment (JackEAllen) [inline queue_processors/queue_processor/queue_processor.py]: Nit: Would this be worth placing in a separate method?

**Comment by ElliotAOram** [comment]
@JackEAllen this is ready for re-review. Thanks

**Comment by JackEAllen** [comment]
Looks good! 👍

**Pull request merged**